### DISCUSSION TITLE
Can I use Toltec with this on software version 3.5

### DISCUSSION THREAD

**Discussion by lm41** [timower/rM2-stuff#26] [Q&A] [answered]
The toltec website says that toltec does not work with os versions above 2.15.1.1189. If I have these patches, will it be possible to install toltec with version 3.5?

**Comment by timower** [comment, marked as answer]
It's unsupported (so don't bother the toltec maintainers if something breaks), and as always if something breaks you'll have to use https://github.com/ddvk/remarkable2-recovery.
But I'm on version 3.5 with toltec & working rm2fb from this repo. If you encounter any problems, feel free to open an issue.

**Reply by lm41** [reply]
Thank you.
